src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 endpoint.userSearch=/v1/users/uuidBySlack
-user.baseURL=https://users
\ No newline at end of file
+endpoint.addKeeper=/keepers
```

Comment:
Shuld we need keepers.baseURL? 
Can only leave the endpoint? keepers.baseURL=http://v1/keepers? What should I use HTTP or HTTPS?